src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/scalable/LabeledVariantAnnotationsWalker.java

```diff
@@ -222,9 +218,11 @@ public void onTraversalStart() {
                     LabeledVariantAnnotationsData.SNP_LABEL));
         }
 
+        useASAnnotations = isAlleleSpecificAnnotationRequested();
+
         if (useASAnnotations && resourceMatchingStrategy != ResourceMatchingStrategy.START_POSITION_AND_MINIMAL_REPRESENTATION) {
-            logger.warn(String.format("The %s argument is ignored when %s is set to true. The START_POSITION_AND_MINIMAL_REPRESENTATION strategy will be used.",
-                    RESOURCE_MATCHING_STRATEGY_LONG_NAME, USE_ALLELE_SPECIFIC_ANNOTATIONS_LONG_NAME));
+            logger.warn(String.format("The %s argument is ignored when allele-specific annotations are requested. The START_POSITION_AND_MINIMAL_REPRESENTATION strategy will be used.",
```

Comment:
I see your point, but I think it can be argued that the user still has to actively request allele-specific annotations via `-A` arguments. Will leave this and below as is, if that's OK!